build.gradle

```diff
@@ -1,19 +1,21 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        gradlePluginPortal()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:4.1.2'
         classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1'
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
+        classpath 'com.vanniktech:gradle-maven-publish-plugin:0.13.0'
     }
 }
 
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        gradlePluginPortal()
```

Comment:
Is `gradlePluginPortal()` necessary here?

Reply:
I'd also vote to keep this as is and address this in another PR when new version needs to be published.

My reasoning is that since `trove4j` is a transitive dependency for Android Build Tools, it might be removed in the next release version for the build tools as mentioned or be moved to Maven Central as well since JCenter will shut down. 

I feel the work around would potentially hide what happened to this. 

I'll add a `TODO` comment on the repository declaration as a note for us when we publish and it breaks.